app/src/common/shared/org/mozilla/vrbrowser/telemetry/GleanMetricsService.java

```diff
@@ -73,13 +73,13 @@ public static void init(@NonNull Context aContext, @NonNull Client client) {
                 Configuration.DEFAULT_TELEMETRY_ENDPOINT,
                 BuildConfig.BUILD_TYPE);
         Glean.INSTANCE.initialize(aContext, true, config);
+        setStartupMetrics();
     }
 
     // It would be called when users turn on/off the setting of telemetry.
     // e.g., SettingsStore.getInstance(context).setTelemetryEnabled();
     public static void start() {
         Glean.INSTANCE.setUploadEnabled(true);
-        setStartupMetrics();
```

Comment:
The only startup metric you're setting is the `channel_name`. I remember discussing this in #1854, but I don't remember the details and I can't find it from a quick skim of the old PR. Why do we need this metric at all? Can't we use the [channel field from `Configuration`](https://mozilla.github.io/glean/javadoc/glean/mozilla.telemetry.glean.config/-configuration/channel.html)?

Reply:
That channel field is used for us to record the build is for `debug` or `release`.